src/js/actions.js

```diff
@@ -0,0 +1,28 @@
+export {actionBtn};
```

Comment:
1. Declare function before exports. 
2. You should export class with methods for every action.